### DISCUSSION TITLE
Mengubah Data Time dari Character menjadi Time

### DISCUSSION THREAD

**Discussion by Kukuhws** [teamalgoritma/community#237] [Q&A] [answered]
Dear Team Algoritma

Mohon bantuannya, saya mau merubah table 'Time' yang sebelumnya character menjadi date tetapi secara bentuk formatnya sama dengan yang sekarang hanya tipe datanya saja yang berubah.

Code yang saya coba, sesuai cheatset lubridate:
spm$Time <- hm(spm$Time)

tapi hasil yang dihasilkan justru 13H 8M 0S, apakah secara formatnya tetap 13:08? 

![image](https://user-images.githubusercontent.com/107305755/182276317-46cd1fb0-1014-437b-864b-9ab331e6e3ca.png)

**Comment by inytss** [comment, marked as answer]
Dear @Kukuhws ,
Untuk mengubah sebuah data waktu dengan tipe data "waktu" Bapak dapat memanfaatkan fungsi `times()` dari package `chron`. Menggunakan fungsi `times()` Bapak dapat menyimpan data waktu dalam format `%H:%M` seperti format data yang sudah ada namun dengan tipe data time. 

Berikut contohnya:
```
spm$Time <- chron::times(spm$Time)
```
Hasil dari output tersebut yaitu kolom `Time` akan terubah tipe datanya menjadi time. Berikut link yang dapat dijadikan bahan bacaan https://stackoverflow.com/questions/22659947/r-how-to-handle-times-without-dates

Semoga membantu. Apabila jawaban ini dirasa membantu, silahkan tekan tombol _"Mark as answer"_.
